docs/api-tutorials/custom-reporter.md

```diff
@@ -0,0 +1,113 @@
+Mocha allows you to define and use custom reporters install via `npm`.
+
+For example, if `mocha-foo-reporter` was published to the npm registry, you could install it via `npm install mocha-foo-reporter -D`, then use it via `mocha --reporter mocha-foo-reporter`.
```

Comment:
How about we use `--save-dev` rather than `-D`. All npm users don't familiar with `-D` flag and it is hard to search even if I also use `-D`.

Reply:
Agreed. I've never used the single letter flag.